packages/shared/src/types/prices.ts

```diff
@@ -1,17 +1,15 @@
 import { CountryGroupId } from '../lib';
 
-interface ProductPriceData {
-    Monthly: {
+export type RatePlan = 'Monthly' | 'Annual';
+type ProductPriceData = {
+    [ratePlan in RatePlan]: {
         price: string;
     };
-    Annual: {
-        price: string;
-    };
-}
-interface CountryGroupPriceData {
-    GuardianWeekly: ProductPriceData;
-    Digisub: ProductPriceData;
-}
+};
+export type GuardianProduct = 'GuardianWeekly' | 'Digisub';
+type CountryGroupPriceData = {
+    [product in GuardianProduct]: ProductPriceData;
+};
 export type Prices = {
     [country in CountryGroupId]: CountryGroupPriceData;
 };
```

Comment:
To be aware - when adding prices template functionality in support-admin-console, I copied over the existing Prices types over to the `bannerVariantPreview.tsx` file. Not an ideal solution as changes in the model in this repo (which should be the Single Source of Truth) may need to be copied across to SAC to keep the Live Preview functionality working.

Reply:
Thanks - it's good to keep these in sync, I'll do a PR on SAC